pulsar-client-admin-api/src/main/java/org/apache/pulsar/common/policies/data/Policies.java

```diff
@@ -125,6 +125,12 @@
     @SuppressWarnings("checkstyle:MemberName")
     public String resource_group_name = null;
 
+    /**
+     * if writer_namespace is set, it means this is a readonly namespace. All data is read from writer_namespace
+     */
+    public String writerNamespace = null;
```

Comment:
I believe that this is a little bit confusing an error prone.
can we add a "readonlyModeEnabled" flag ? (or with a better name)

Reply:
@eolivelli Thank you for pointing this out, totally agree with you.
Naming is always the difficult part.  'Readonly' is a little confusing in this project as there is already a 'ReadonlyManagedLedger', but it's completely different aspect of readonly.

Added "shadowReaderModeEnabled" for this flag. Is that ok?